packages/test/test-gc-sweep-tests/src/dataObjectWithChildDataObject.ts

```diff
@@ -4,16 +4,13 @@
  */
 
 import { DataObjectFactory } from "@fluidframework/aqueduct";
-import { assert } from "@fluidframework/common-utils";
-import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { assert, delay } from "@fluidframework/common-utils";
 import { SharedCounter } from "@fluidframework/counter";
 import { DataObjectWithCounter, dataObjectWithCounterFactory } from "./dataObjectWithCounter";
 
 export class RootDataObjectWithChildDataObject extends DataObjectWithCounter {
```

Comment:
The more doc comments on classes and functions the better for someone trying to ramp up on the code/structure!